### ISSUE TITLE
exporter leaking file descriptors

### ISSUE THREAD

**Issue by tommywo** [ribbybibby/ssl_exporter#30]
After some time ssl_exporter is unable to scrape targets.
```
time="2020-06-04T11:39:37Z" level=error msg="Get https://example.com:443: dial tcp: lookup example.com on 8.8.8.8:53: dial udp 8.8.8.8:53: socket: too many open files" source="ssl_exporter.go:108"
```
Similar issue happend in different exporter: https://github.com/bosh-prometheus/shield_exporter/issues/3 , fixed in pull request: https://github.com/shieldproject/shield/pull/280/files

**Comment by tommywo** [comment]
Yes. Thank you :)

**Comment by ribbybibby** [comment]
Fixed in v1.0.1
